src/repertoire/config.py

```diff
@@ -72,6 +142,40 @@ class Config(RepertoireSettings):
         validation_alias=AliasChoices("REPERTOIRE_TOKEN", "token"),
     )
 
+    schema_version: str | None = Field(
+        None,
+        title="Default schema version",
+        description="Schema version to use",
+    )
+
+    schema_source_template: str | None = Field(
+        None,
+        title="URL template for schema downloads",
+        description=(
+            "Template for schema download URLs."
+            "Examples: 'gs://bucket/{version}.tar.gz'"
+        ),
+    )
+
+    tap_schema_extensions_path: str | None = Field(
+        None,
+        title="Path to TAP_SCHEMA extensions",
+        description=(
+            "YAML file with TAP_SCHEMA extension columns. "
+            "Defaults to felis resource if not set."
+        ),
+    )
+
+    tap_servers: dict[str, TapServerConfig] = Field(
+        default_factory=dict,
+        title="TAP schema configuration by application",
+        description=(
+            "Configuration for TAP servers whose TAP_SCHEMA tables are "
+            "managed by repertoire. Keys are application names "
+            "(tap, ssotap, etc)."
+        ),
+    )
```

Comment:
I think we should nest all of these under a new `tap` key (so put them in a separate model), and then you can drop the `tap_` prefix added to some of the configuration options. That makes it more obvious that all of these settings are only about TAP, not any other type of schema. That may also simplify some of the arguments to the TAP schema service, since you can just pass in the full TAP configuration.